### ISSUE TITLE
[Feature Request] Live Activity Support

### ISSUE THREAD

**Issue by BambuLabDiscord** [bambulab/Bambu-Handy#36]
add a live activity for ios that allows the printers status from the lock screen or from the dynamic island.

**Comment by pepsiliu** [comment]
That's a very good interaction, we'll take it to our feature list.

**Comment by freeformz** [comment]
+1 https://forum.bambulab.com/t/feature-request-support-live-activities-on-ios/19747

**Comment by Josh4532** [comment]
+1 this would be cool

**Comment by stef1949** [comment]
+1 any update for this?

**Comment by stef1949** [comment]
+1 any update for this?

**Comment by cfuller007** [comment]
I would love the same along with support to Dynamic Island. I just got my first Bambu lab printer and I really missed that from OctoPod.

**Comment by freeformz** [comment]
I started using Joint Craft for this: https://apps.apple.com/us/app/joint-craft-3d-print-monitor/id6455888249

**Comment by stef1949** [comment]
> I started using Joint Craft for this: https://apps.apple.com/us/app/joint-craft-3d-print-monitor/id6455888249

Agreed. I would have been nice for Bambu to implement this but Joint Craft just provides a better user experience at the moment :/

**Comment by stef1949** [comment]
> That's a very good interaction, we'll take it to our feature list.

Any update on this?

@pepsiliu @LibaoBambu

**Comment by Alshootfa** [comment]
+1 from me too. This feature is probably the most needed feature for Bambu Handy in my point of view.

**Comment by MilkMGN** [comment]
+1 Something Octopod looking would be nice. Even an Apple watch inrtegration too

**Comment by StevenLi-phoenix** [comment]
@BambuLabDiscord @pepsiliu 
This issue could be closed now. It is implemented in 2.19.0
https://apps.apple.com/us/app/bambu-handy/id1625671285
```
- Added support for Live Activities to show print progress on ios
- 增加了对 Live Activities 的支持，以在 iOS 上显示打印进度
```

**Comment by stef1949** [comment]
Thank you!!

**Comment by stef1949** [comment]
Sorry to be a pain, but live activities don't seem to be working :/

**Comment by StevenLi-phoenix** [comment]
seems hot fix broke it again ...

**Comment by stef1949** [comment]
@LibaoBambu 💖

**Comment by StevenLi-phoenix** [comment]
Live activities only works if you open the app while printing.
And it works now
Thank you devs

**Comment by AndaleR** [comment]
I was looking for that a long time. And then I found this point and I needed to activate it. And restart the app. NOW it works.![image](https://github.com/user-attachments/assets/778a145c-82bc-412c-82f9-84506081a404)
